test/test_model.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 
 datafiles_path = os.path.join('test', 'data')
-
+print(datafiles_path)
```

Comment:
is this just print debug, or should this use `logging`

Reply:
it was a print debug. removed it.